.github/workflows/backport.yml

```diff
@@ -0,0 +1,37 @@
+on:
+  push:
+    branches:
+       - master
+
+jobs:
+  pretest:
+    runs-on: ubuntu-18.04
+    env:
+      CUPY_CI: GitHub
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+
+    - name: set ssh
+      uses: webfactory/ssh-agent@v0.5.0
+      with:
+          ssh-private-key: ${{secrets.SSH_PRIVATE_KEY}}
+
+    - name: setup python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.7'
+
+    - name: setup backport
+      run: |
+        pip install pygithub
+        git config --global user.name "CuPy Automatic Backport"
+        git config --global user.email "33715081+chainer-ci@users.noreply.github.com"
+        git clone https://github.com/cupy/backport.git
+
+    - name: run_backport
+      run: |
+        cd backport
+        python backport.py --repo cupy --sha ${{github.event.after}} --token ${{secrets.BACKPORT_TOKEN}}
```

Comment:
Could you use `BACKPORT_GITHUB_TOKEN` environment variable?

Reply:
changed! :)